cluster/addons/dns/coredns/coredns.yaml.sed

```diff
@@ -116,7 +116,7 @@ spec:
         beta.kubernetes.io/os: linux
       containers:
       - name: coredns
-        image: k8s.gcr.io/coredns:1.5.0
+        image: k8s.gcr.io/coredns:$CORE_DNS_VERSION
```

Comment:
Based on changes in https://github.com/kubernetes/kubernetes/pull/78691 I think the `readinessProbe.httpGet` fields should also be modified

Reply:
[I agree](https://github.com/kubernetes/kubernetes/pull/78851#issuecomment-500429485), and the ConfigMap Corefile needs to include the ready plugin when being deployed in 1.5.0.